demo/component/LineChart.tsx

```diff
@@ -416,7 +425,7 @@ export default class Demo extends Component<any, any> {
   };
 
   render() {
-    const { data, data01, data02, opacity } = this.state;
+    const { data, data01, data02, data04, opacity } = this.state;
```

Comment:
Nah we shouldn't keep maintaining the demo. This was just completely mimicking the linked PR which was pre-storybook. 


I think it's fine to merge changes to the demo since it doesn't hurt anything, but overall yeah we have no need for it anymore.